spec/controllers/tickets_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe TicketsController, type: :request do
+  let(:admin) { create(:admin, email: 'admin@wikiedu.org') }
+  let(:user) { create(:user, email: 'student@hello.edu') }
+
+  let(:ticket) do
+    TicketDispenser::Dispenser.call(
+      content: 'hello',
+      owner_id: admin.id,
+      sender_id: user.id
+    )
+  end
+  let(:message) { ticket.messages.first }
+
+  before do
+    login_as admin
+  end
+
+  describe '#dashboard' do
+    it 'renders the main tickets dashboard URL' do
+      get '/tickets/dashboard'
+      expect(response.status).to eq(200)
+      expect(response.body).to include("id='react_root'")
+    end
+
+    it 'renders an arbitrary tickets path' do
+      get '/tickets/foo'
+      expect(response.status).to eq(200)
+      expect(response.body).to include("id='react_root'")
```

Comment:
This is a bit of an odd test. We're basically just checking that this route exists via React Router, right? If so, it may be useful to just rename this to something more like "has a tickets 'show' path" or something more direct.

Reply:
Yeah, I guess testing for a real show path makes more sense.